cfme/tests/services/test_orchestration_template.py

```diff
@@ -77,9 +84,10 @@ def test_name_required_error_validation(provisioning):
     template = OrchestrationTemplate(template_type=template_type,
                                      template_name=None,
                                      description="my template")
+    method = METHOD_TORSO.replace('CloudFormation', random_desc())
 
     with error.expected(flash_msg):
-        template.create(METHOD_TORSO)
+        template.create(method)
```

Comment:
Thinking about this, why not create a function scoped fixture that would return a ```METHOD_TORSO``` with a randomized string in the description? That way you have eliminated the duplicate random calls, and making if function scoped will mean you don't need to worry about them colliding.